packages/azure-http-specs/specs/azure/resource-manager/common-properties/mockapi.ts

```diff
@@ -126,3 +126,60 @@ Scenarios.Azure_ResourceManager_CommonProperties_ManagedIdentity_updateWithUserA
     },
     kind: "MockApiDefinition",
   });
+
+Scenarios.Azure_ResourceManager_CommonProperties_Error_get = passOnCode(404, {
+  uri: "/subscriptions/:subscriptionId/resourceGroups/:resourceGroup/providers/Azure.ResourceManager.CommonProperties/confidentialResources/:resourceName",
+  method: "get",
+  request: {
+    params: {
+      subscriptionId: SUBSCRIPTION_ID_EXPECTED,
+      resourceGroup: RESOURCE_GROUP_EXPECTED,
+      resourceName: "confidential",
+      "api-version": "2023-12-01-preview",
+    },
+    status: 404,
```

Comment:
Is it intentional that we define a field `request.status` with an expected response status code?